packages/vscode-plugin/README.md

```diff
@@ -1,30 +1,38 @@
 # Harper for VS Code
 
-Currently, `harper-ls` needs to be in your `PATH` for this extension to work. [Read here to see how to install it](/harper-ls/README.md#installation).
-
-## Manually Packaging and Installing
-
-### Requirements
-
-- [`yarn`](https://classic.yarnpkg.com/en), as the package manager
-- [`just`](https://just.systems), as the command runner
-
-### Steps
-
-1. Clone or download the Harper repository:
-
-   ```console
-   git clone https://github.com/elijah-potter/harper && cd harper
-   ```
-
-2. Package the extension:
-
-   ```console
-   just package-vscode
-   ```
-
-3. Install the extension:
-
-   ```console
-   code --install-extension packages/vscode-plugin/harper-0.0.1.vsix
-   ```
+Harper is the grammar checker for developers. It checks for spelling and grammar errors in your Markdown files and code comments. You can find out more by checking the [project's readme](/README.md) or by visiting the [website](https://writewithharper.com).
+
+## Installation
+
+## Features
+
+### Commands
+
+| Command                         | Id                              | Description         |
+| ------------------------------- | ------------------------------- | ------------------- |
+| Harper: Restart Language Server | `harper.languageserver.restart` | Restart `harper-ls` |
+
+### Settings
```

Comment:
Is it necessary to include linter descriptions in the README? I would imagine they should be self-explanatory enough to stand alone in the settings page. I'm not a huge fan of the duplication with `package.json`.

Reply:
I understand. I'd also prefer not to duplicate them, but I just thought it might help people who find out about Harper from the VS Code Marketplace first to get an overview of what exactly it checks for. However, I can remove it if you think it is unnecessary.